### PR TITLE
Suggested styles

### PR DESCRIPTION
**Pull request by AaronRutley**
I've suggested some styles to improve the 'gutenberg-demo' post.

### CODE REVIEW

**Review by ntwb — APPROVED** [review]
LGTM

File: style.css [hunk 1]
```diff
@@ -819,3 +819,7 @@ object {
 .wp-block-embed {
 	text-align: center;
 }
+
+.wp-block-embed .twitter-tweet-rendered { 
+	margin:0 auto;
```
Comment (tyrann0us) [inline style.css]: Nit: Missing whitespace between property and value?

File: blocks.css [hunk 2]
```diff
@@ -32,7 +33,13 @@ ol,
...
 .wp-block-gallery.alignfull,
-..wp-block-text-columns.alignfull{
+.wp-block-text-columns.alignfull{
   margin: 1.5em 0;
   max-width: 100%;
 }
+
+.wp-block-button.aligncenter {
+	display: block;
+	margin:0 auto;
```
Comment (tyrann0us) [inline blocks.css:43]: Nit: Missing whitespace between property and value?

**Review by tyrann0us — COMMENTED** [review]
Nit: Missing whitespace.

File: blocks.css (hunk 2 above)
Comment (ntwb) [reply blocks.css:43]: Ha, I missed that 😏

Comment (melchoyce) [reply blocks.css:43]: More indentation here than in the rest of the stylesheet — maybe swap out to match the rest of the stylesheet?

File: blocks.css [hunk 3]
```diff
@@ -30,7 +33,13 @@ ol,
...
+	text-align: center;
```
Comment (melchoyce) [inline blocks.css:44]: More indentation here than in the rest of the stylesheet — maybe swap out to match the rest of the stylesheet?

File: style.css [hunk 4]
```diff
@@ -815,3 +815,11 @@ object {
 .gallery-caption {
   display: block;
 }
+
+.wp-block-embed {
+	text-align: center;
+}
+
+.wp-block-embed .twitter-tweet-rendered { 
+	margin:0 auto;
```
Comment (melchoyce) [inline style.css:824]: Needs a space between `margin` and `0`

File: blocks.css [hunk 5]
```diff
@@ -30,7 +33,13 @@ ol,
 .wp-block-cover-text.alignfull,
 .wp-block-image.alignfull,
 .wp-block-gallery.alignfull,
-..wp-block-text-columns.alignfull{
+.wp-block-text-columns.alignfull{
```
Comment (melchoyce) [inline blocks.css:36]: Needs a space between `.alignfull` and `{`

File: blocks.css (hunk 2 above)
Comment (melchoyce) [reply blocks.css:43]: Looking at the demo, I don't think this should be full-width.

**Review by melchoyce — CHANGES_REQUESTED** [review]
Some minor spacing issues, but otherwise looking good — if you patch those up, and fix up the button width, I think it'll be good to go.

**Comment by karmatosed** [comment]
Can you redo this PR with the suggestions in place please @AaronRutley? Note, there have been updates to the theme so make sure you grab the latest and resolve conflicts. Thanks!

**Comment by karmatosed** [comment]
Closing as we have conflicts and have split this from the .org site now.

**Pull request closed without merging**